homeassistant/components/device_tracker/asuswrt.py

```diff
@@ -118,11 +119,10 @@ def __init__(self, config):
         if self.protocol == 'ssh':
             self.connection = SshConnection(
                 self.host, self.port, self.username, self.password,
-                self.ssh_key, self.mode == 'ap')
+                self.ssh_key)
```

Comment:
ap is still used, look at ```get_asuswrt_data()``` please revert this

Reply:
@kennedyshead That method uses only the `mode` attribute of AsusWrtDeviceScanner.  The `ap` parameter of SshConnection (and TelnetConnection) is only used to set the attribute `SshConnection._ap`. This is not needed and doesn't make sense as a property of the connection anyway.